http/src/test/java/com/gitthub/awsjavakit/http/OAuthResponse.java

```diff
@@ -0,0 +1,10 @@
+package com.gitthub.awsjavakit.http;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record OAuthResponse(
```

Comment:
This class is used only in testing (and it is in the "test" sources). Probably we don't need it.